apps/client/src/shared/pages/loading/loading.css.ts

```diff
@@ -0,0 +1,7 @@
+import { style } from '@vanilla-extract/css';
+import { themeVars } from '@confeti/design-system/styles';
+
+export const loadingSection = style({
+  ...themeVars.display.flexCenter,
+  height: 'calc(100vh - 5rem)',
```

Comment:
```suggestion
  height: 'calc(100dvh)',
```
https://abcdqbbq.tistory.com/104

Reply:
헉... 또 하나 배워갑니다 아티클 바로 저장 ~

Reply:
헉 저도 배워갑니다 ~